modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/Utils/KubernetesProvisioningUtils.java

```diff
@@ -119,8 +121,8 @@ public static HttpClient getHttpClientForKubernetes()
 
     public static Namespace getNameSpace(ApplicationContext applicationContext) {
         Namespace namespace = new Namespace();
-        namespace.getMetadata()
-                .setNamespace(applicationContext.getTenantDomain() + "-" + applicationContext.getCurrentStage());
+        namespace.getMetadata().setNamespace(
+                applicationContext.getTenantInfo().getTenantDomain() + "-" + applicationContext.getCurrentStage());
```

Comment:
This is fixed in following branch.
https://github.com/wso2/product-af/blob/k8s-Integration/modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/Utils/KubernetesProvisioningUtils.java#L71